app/lib/shared/storage.dart

```diff
@@ -33,6 +33,15 @@ extension BucketExt on Bucket {
       rethrow;
     }
   }
+
+  Future uploadPublic(Bucket bucket, String objectName, int length,
```

Comment:
Why bucket as argumnet? this is an extension method right? shouldn't it just use `this`?

Reply:
Done, you are right.